ensemble/ctf/gaussian_function_component.py

```diff
@@ -18,7 +18,7 @@
 POINTER_MAP = {'move': 'hand', 'resize': 'size left'}
 MIN_GAUSSIAN_STD = 2.0
 GAUSSIAN_RADIUS_STD_SCALE = 50.0
-MAX_GAUSSIAN_NUM_SAMPLES = 256
+MAX_NUM_SAMPLES = 256
```

Comment:
Looks like `MIN_GAUSSIAN_STD` and `GAUSSIAN_RADIUS_STD_SCALE` can be removed now, no?